src/shared/regexps.ts

```diff
@@ -1,3 +1,13 @@
+// EVM regexps
 export const RE_ADDRESS = /0x[0-9a-f]{40}\b/i;
```

Comment:
Should probably rename this to `RE_EVM_ADDRESS` for clarity.

Reply:
Addressed in https://github.com/rolod0x/rolod0x/pull/490/commits/d5db96d1a9a550dc39d8c31a82100ca86634e708